packages/amplify-cli/src/telemetry/config.ts

```diff
@@ -0,0 +1,27 @@
+import fs from 'fs-extra';
+import uuid from 'uuid';
+
+export class Config {
+  installationUuid: String;
+  isUsageTrackingEnabled: boolean;
+  constructor() {
+    this.installationUuid = uuid.v4();
+    this.isUsageTrackingEnabled = true;
+  }
+
+  static get(path: string): Config {
```

Comment:
As we agreed on that this config should be in an amplify level configuration file with a `usageData` section the reading/writing of that file should not be a telemetry concern but shared code should be used for that.